README.md

```diff
@@ -387,6 +387,13 @@ The IP configuration supports the following options:
     When using a caching DNS plugin (dnsmasq or systemd-resolved in NetworkManager.conf)
     then "edns0" and "trust-ad" are automatically added.
 
+- `gateway4` and `gateway6`
+
+    The gateway associated with the IPv4 configuration and IPv6 configuration. This is
+    only meaningful if the corresponding IPv4 addresses or IPv6 addresses is also set.
+    The gateway's main purpose is to control the next hop of the standard default route
```

Comment:
If you write "main purpose" it implies that there are other purposes. What other purposes are there?

Reply:
"main purpose" removed.